backend/src/graphql/resolvers/ContactFormResolver.ts

```diff
@@ -0,0 +1,22 @@
+import { Resolver, Mutation, Query, Arg } from 'type-graphql'
+
+import { ContactFormInput } from '#inputs/ContactFormInput'
+import { prisma } from '#src/prisma'
+
+@Resolver()
+export class ContactFormResolver {
+  @Mutation(() => Boolean)
+  async createContactForm(
+    @Arg('contactFormData') contactFormData: ContactFormInput,
+  ): Promise<boolean> {
+    await prisma.contactForm.create({ data: contactFormData })
+    // code to send email goes here
+    return true
+  }
+
+  // needed to avoid: GraphQLError: Type Query must define one or more fields
+  @Query(() => Boolean)
+  contactForm(): boolean {
+    return true
+  }
```

Comment:
```suggestion
  // needed to avoid: GraphQLError: Type Query must define one or more fields
  @Query(() => Boolean)
  contactForm(): boolean {
    return true
  }
```
Why do you need to define it in the first place?

Reply:
I need at least one QUERY to have a valid schema.

Reply:
Is this prisma related, graphql or type-graphql?
Does this mean every model needs a query or just one global one?

Reply:
I understand you need at least on query in the whole schema.